autoload/which_key.vim

```diff
@@ -272,17 +261,14 @@ function! s:getchar() abort
     return '<CR>'
   endif
 
-  let input .= which_key#char_handler#parse_raw(c)
-  if s:has_children(input)
```

Comment:
This loop was introduced to distinguish the mappings like `gc` and `gcc` as initially reported in #3, I guess it's still useful.